core/sequencer/server.go

```diff
@@ -68,13 +68,13 @@ func createMetrics(mf monitoring.MetricFactory) {
 		domainIDLabel)
 }
 
-// Queue reads messages that haven't been deleted off the queue.
-type Queue interface {
-	// HighWatermarks returns the highest primary key for each shard in the mutations table.
+// LogsReader reads messages in multiple log logs.
+type LogsReader interface {
+	// HighWatermarks returns the highest primary key for each log in the mutations table.
```

Comment:
We've been deliberately vague here. 
The PK can either be a timestamp - the current, tabled based implementation - but it could also be an index in a Trillian Log when we move to that in the future. 

The difference to the caller, though, should be completely transparent.

Reply:
Ok, maybe "identifier" or similar, then?